src/primitives/felt.test.ts

```diff
@@ -1,20 +1,24 @@
 import { test, expect, describe } from 'bun:test';
 import { Felt } from './felt';
-import { OutOfRangeBigInt, PrimitiveError } from 'errors/primitives';
-import { Relocatable } from './relocatable';
 
 describe('Felt', () => {
   describe('constructor', () => {
-    test('should throw if initializing a felt with a negative inner', () => {
-      expect(() => new Felt(-10n)).toThrow(
-        new PrimitiveError(OutOfRangeBigInt)
+    test('should properly initialize a Felt with a negative bigint as parameter', () => {
+      const negativeUnderPrime = -(Felt.PRIME + 10n);
+      const negativeEvenUnderPrime = -(Felt.PRIME + Felt.PRIME + 15n);
```

Comment:
I don't understand these choices. Why not just `-10` or `-10n`?
+ never do two tests in the same test
+ even is "pair" while you use `-15`

Reply:
I wanted to check that for `n = -(q*Felt.PRIME + r)`, with `q,r > 0`, the wrapping works as expected, but it isn't that meaningful. I'll stick to `-10n`

`EvenUnder` is bad wording, it wasn't about the parity